client/src/lib/schemas/permission.schema.ts

```diff
@@ -9,7 +9,7 @@ export const permissionNameSchema = z.enum([
   "ASSIGN_ROLE",
   "DELETE_USER",
   "LOGIN",
-  "REGISTER",
+  "REGISTER", // Đăng kí gì ?
```

Comment:
It's placeholder for permission, please don't add comment here to avoid conflict